Binary files orig/flowWorkspace/src/.Makevars.ucrt.swp and patched/flowWorkspace/src/.Makevars.ucrt.swp differ
diff -Nru orig/flowWorkspace/src/Makevars.ucrt patched/flowWorkspace/src/Makevars.ucrt
--- orig/flowWorkspace/src/Makevars.ucrt	2024-10-24 02:00:12.000000000 +0200
+++ patched/flowWorkspace/src/Makevars.ucrt	2025-03-04 12:32:07.884836900 +0100
@@ -7,7 +7,7 @@
 #needs to wrap in $(shell) to strip the quotes returned by rhdf5lib::pkgconfig
 RHDF5_LIBS= $(shell "${R_HOME}/bin/Rscript" -e "Rhdf5lib::pkgconfig('PKG_CXX_LIBS')")
 
-PKG_LIBS += $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS) `${R_HOME}/bin/Rscript -e "cytolib:::cytolib_LdFlags();cat(' ');RProtoBufLib:::LdFlags();cat(' ');RcppParallel::RcppParallelLibs()"` ${RHDF5_LIBS} -lws2_32  -lbcrypt
+PKG_LIBS += $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS) $(shell "${R_HOME}/bin/Rscript" -e "cytolib:::cytolib_LdFlags();cat(' ');RProtoBufLib:::LdFlags();cat(' ');RcppParallel::RcppParallelLibs()") ${RHDF5_LIBS} -lws2_32  -lbcrypt
 
 USERDIR = ${R_PACKAGE_DIR}/lib${R_ARCH}
 PKGLIB = ${USERDIR}/libflowWorkspace.a
